src/coreclr/vm/exkind.h

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// RuntimeExceptionKind.h
+//
+
+#ifndef __exkind_h__
+#define __exkind_h__
+
+#include <cstdint>
```

Comment:
```suggestion
#include <cstdint>
#include <functional>
```
fixes the linux build include ordering issue, but you can also add it to `vm/common.h` around line 60 as this header isn't using template.